src/main/java/com/dnastack/ga4gh/dataconnect/adapter/trino/ThrowableTransformer.java

```diff
@@ -28,7 +28,9 @@
         entry(TrinoInsufficientResourcesException.class, HttpStatus.SERVICE_UNAVAILABLE),
         entry(QueryParsingException.class, HttpStatus.BAD_REQUEST),
         entry(InvalidQueryJobException.class, HttpStatus.BAD_REQUEST),
-        entry(UnexpectedQueryResponseException.class, HttpStatus.INTERNAL_SERVER_ERROR));
+        entry(UnexpectedQueryResponseException.class, HttpStatus.INTERNAL_SERVER_ERROR),
+        entry(TrinoUserUnauthorizedException.class, HttpStatus.UNAUTHORIZED),
+        entry(TrinoUserForbiddenException.class, HttpStatus.FORBIDDEN));
```

Comment:
When do these get used? I see that we are doing error handling in `toTableData`.

Reply:
Basically when I traced the code, I saw that the exceptions that are thrown in `toTableData` are mapped to their proper HTTP response codes using this map (there is a method further down in this class that fetches the status code for a given exception).